lib/postprocess/image.js

```diff
@@ -100,13 +100,10 @@ module.exports = async files => {
+          imageExtensions = 'png'
+          imageBuffer = Buffer.from(imageSrc, 'base64')
+          console.warn('Image from ' + imageConfig.src + ' throws the following error: ' + e)

```

Comment:
this gives nice warnings in the console, but how can I make them show up in CloudWatch?